lib/pf/Switch/Meraki/AP_http_V2.pm

```diff
@@ -0,0 +1,380 @@
+package pf::Switch::Meraki::AP_http_V2;
+
+=head1 NAME
+
+pf::Switch::Meraki::AP_http_V2
+
+=head1 SYNOPSIS
+
+The pf::Switch::Meraki::AP_http_V2 module implements an object oriented interface to
+manage the external captive portal on Meraki access points
+
+=head1 STATUS
+
+Developed and tested on a MR12 access point
+
+=head1 BUGS AND LIMITATIONS
+
+=head2 client IP cannot be computed from parseUrl
+
+The Meraki sends a NATed IP address in the URL even though the client is bridged.
+The only workaround is to have the DHCP traffic forwarded to PacketFence.
+
+=cut
+
+use strict;
+use warnings;
+
+use base ('pf::Switch');
+
+use Net::SNMP;
+use Try::Tiny;
+
+use pf::Switch::constants;
+use pf::constants;
+use pf::web::util;
+use pf::config;
+use pf::roles::custom;
+use pf::util;
+use pf::node;
+use pf::util::radius qw(perform_coa perform_disconnect);
+use pf::node qw(node_view);
+use pf::violation;
+use pf::locationlog;
+
+
+=head1 SUBROUTINES
+
+=cut
+
+# CAPABILITIES
+# access technology supported
+sub description { 'Meraki cloud controller v2' }
+sub supportsWirelessMacAuth { return $TRUE; }
+sub supportsExternalPortal { return $TRUE; }
+sub supportsWirelessDot1x { return $TRUE; }
+sub supportsWebFormRegistration { return $FALSE }
+
+=head2 parseUrl
+
+This is called when we receive a http request from the device and return specific attributes:
+
+client mac address
+SSID
+client ip address
+redirect url
+grant url
+status code
+
+=cut
+
+sub parseUrl {
```

Comment:
I don't think that this is even used since you do the flow with the session id.

Can you double check this is of any use in this module